zipkin-server/README.md

```diff
@@ -213,223 +223,330 @@ $ java -jar zipkin.jar
+We strongly recommend that you read more about these configurations from ElasticSearch's official documentation since they directly impact the performance of ElasticSearch.
 
-The above properties allow the most common SSL setup to work out of box. If you need more
-customization, please make a comment in [this issue](https://github.com/openzipkin/zipkin/issues/2774).
+### BanyanDB storage components
 
-#### Automatic Index Creation
-Zipkin will automatically create new indices as needed. Elasticsearch by default [allows](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html#index-creation) automatic creation of said indices, though your local install may have been configured to disallow it. You can verify this in the cluster settings: `action.auto_create_index: false`.
+[BanyanDB](https://github.com/apache/skywalking-banyandb) is a dedicated storage implementation developed by the SkyWalking Team and the community.
+Currently, BanyanDB is still in the PoC stage and it is not recommended to use it in a production environment.
+
+The following apply when `ZIPKIN_STORAGE` is set to `banyandb`:
+    
+    * `ZIPKIN_STORAGE_BANYANDB_HOST`: The host of BanyanDB. Defaults to `127.0.0.1`.
+    * `ZIPKIN_STORAGE_BANYANDB_PORT`: The port of BanyanDB. Defaults to `17912`.
+    * `ZIPKIN_STORAGE_BANYANDB_MAX_BULK_SIZE`: The max bulk size of BanyanDB. Defaults to `5000`.
+    * `ZIPKIN_STORAGE_BANYANDB_FLUSH_INTERVAL`: The flush interval of BanyanDB. Defaults to `15`.
+    * `ZIPKIN_STORAGE_BANYANDB_METRICS_SHARDS_NUMBER`: The number of shards of metrics index. Defaults to `1`.
+    * `ZIPKIN_STORAGE_BANYANDB_SUPERDATASET_SHARDS_FACTOR`: The shards factor of zipkin span record index. Defaults to `2`.
+    * `ZIPKIN_STORAGE_BANYANDB_CONCURRENT_WRITE_THREADS`: The number of concurrent write threads. Defaults to `15`.
+    * `ZIPKIN_STORAGE_BANYANDB_BLOCK_INTERVAL_HOURS`: The block interval hours of BanyanDB. Defaults to `24` hour.
+    * `ZIPKIN_STORAGE_BANYANDB_SEGMENT_INTERVAL_DAYS`: The segment interval days of BanyanDB. Defaults to `1` days.
+    * `ZIPKIN_STORAGE_BANYANDB_SUPER_DATASET_BLOCK_INTERVAL_HOURS`: The zipkin span record block interval hours of BanyanDB. Defaults to `4` hour.
+    * `ZIPKIN_STORAGE_BANYANDB_SUPER_DATASET_SEGMENT_INTERVAL_DAYS`: The zipkin span record segment interval days of BanyanDB. Defaults to `1` days.
+    * `ZIPKIN_STORAGE_BANYANDB_SPECIFIC_GROUP_SETTINGS`: The specific group settings of BanyanDB. ex, `{"group1": {"blockIntervalHours": 4, "segmentIntervalDays": 1}}`. Defaults to "".
+
+Example usage:
+
+```bash
+$ ZIPKIN_STORAGE=banyandb java -jar zipkin.jar
+```
 
 ### Legacy (v1) storage components
 The following components are no longer encouraged, but exist to help aid
 transition to supported ones. These are indicated as "v1" as they use
 data layouts based on Zipkin's V1 Thrift model, as opposed to the
 simpler v2 data model currently used.
```

Comment:
Do we need this statement? I think the new of v3 includes making MySQL and PG back to be available.

Reply:
Been deleted.